extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1013,6 +1054,101 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return ['https://partner.googleadservices.com',
       'https://tpc.googlesyndication.com'];
   }
+
+  /** @override */
+  getNonAmpCreativeRenderingMethod(headerValue) {
+    if (this.isFluid_) {
+      return XORIGIN_MODE.SAFEFRAME;
+    }
+    return super.getNonAmpCreativeRenderingMethod(headerValue);
```

Comment:
return this.isFluid_ ? XORIGIN_MODE.SAFEFRAME : super...

Reply:
Done.